auth/src/main/java/com/firebase/ui/auth/util/GoogleSignInHelper.java

```diff
@@ -26,7 +25,7 @@ public static GoogleSignInHelper getInstance(FragmentActivity activity) {
 
     public Task<Status> signOut() {
         final TaskCompletionSource<Status> statusTask = new TaskCompletionSource<>();
-        getConnectedApiTask().addOnSuccessListener(new OnSuccessListener<Bundle>() {
+        getConnectedApiTask().addOnSuccessListener(new ExceptionForwarder<Bundle>(statusTask) {
```

Comment:
@samtstern Today's not my day with tasks! 😵 PTAL at 15fbed3, I'm pretty sure that will work.